api/src/main/java/org/apache/iceberg/view/SQLViewRepresentation.java

```diff
@@ -18,14 +18,16 @@
  */
 package org.apache.iceberg.view;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
Can we use `javax.annotation.Nullable` instead?

Reply:
Updated!